Dockerfile.assisted-service-build

```diff
@@ -1,31 +1,31 @@
-FROM quay.io/centos7/postgresql-12-centos7:latest
-
-COPY --from=quay.io/goswagger/swagger:v0.28.0 /usr/bin/swagger /usr/bin/goswagger
-COPY --from=quay.io/edge-infrastructure/swagger-codegen-cli:2.4.18 /opt/swagger-codegen-cli /opt/swagger-codegen-cli
-COPY --from=quay.io/app-sre/golangci-lint:v1.37.1 /usr/bin/golangci-lint /usr/bin/golangci-lint
+FROM quay.io/centos/centos:stream9
```

Comment:
don't forget to add it afterwards to ``release/ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml`` as image substitution

Reply:
The build image shouldn't be part of the CI. It's a huge image.

Reply:
No, you must have the building of assisted-service-build in the CI and keep it "unused". It should not block any kind of job from executing because other jobs are dependent on other images that are being built
Otherwise you have an entire code that is just not tested and still used in local environments

Reply:
don't see why it's needed. If something would break we will get notify by the assisted service contributors but it's not critical

Reply:
Let's talk about it offline
It costs us 0 money and time (because it's not a dependency to any job other than images-build) to build that image, and can save us lots of bugs and frustration. So it's a big deal to me but yet not relevant to this exact PR